### PR TITLE
[5.0] use ComponentMonitoringClient instead of ComponentMonitoringDB

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

FIX: use ComponentMonitoringClient instead of deprecated ComponentMonitoringDB in SystemAdministrationHandler

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review, please

File: src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py [hunk 1]
```diff
@@ -636,101 +633,30 @@ def web_getSelectionData(self):
...
-            componentNames = list(json.loads(self.get_argument("ComponentName")))
-
         componentModules = []
         if "ComponentModule" in self.request.arguments:
             componentModules = list(json.loads(self.get_argument("ComponentModule")))
 
-        showAll = 0
-        if "showAll" in self.request.arguments:
-            showAll = int(self.get_argument("showAll"))
-
```
Comment (TaykYoku) [inline src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py:661]: Do web portal no longer need option "showAll"?
The showAll option is also described in /WebAppDIRAC/WebApp/static/DIRAC/SystemAdministration/classes/SystemAdministration.js

File: src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py [hunk 2]
```diff
@@ -636,101 +633,30 @@ def web_getSelectionData(self):
...
-                            if module != "" and module in componentModules:
-                                fullNames += [path]
-                            elif module == "" and j in componentModules:
-                                fullNames += [path]
 
-                            compMatching[path] = module if module != "" else path
+        condDict = {"Type": componentTypes, "DIRACModule": componentModules}
+        gLogger.debug("condDict" + str(condDict))
 
+        res = ComponentMonitoringClient.getComponents(condDict, True, True)
```
Comment (TaykYoku) [inline src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py]:
```suggestion
        res = ComponentMonitoringClient().getComponents(condDict, True, True)
```

File: src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py [hunk 3]
```diff
@@ -636,101 +633,30 @@ def web_getSelectionData(self):
...
+        if not res["OK"]:
+            self.finish({"success": "false", "error": res["Message"]})
```
Comment (TaykYoku) [inline src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py:653]: If you do not return at this point, the code will continue to run until the end of the function and will fail with `KeyError: 'Value'`

```suggestion
            self.finish({"success": "false", "error": res["Message"]})
            return
```

File: src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py (hunk 1 above)
Comment (fstagni) [reply src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py:661]: I removed it because `showAll` is unused in this module.

**Pull request merged**